src/easylink/implementation_metadata.yaml

```diff
@@ -1,14 +1,14 @@
 step_1_python_pandas:
   steps:
   - step_1
-  image_path: /mnt/team/simulation_science/priv/engineering/er_ecosystem/images/python_pandas.sif
+  image_name: python_pandas.sif
```

Comment:
this is important. We're now using an `image_name` key instead of a fully qualified `image_path`. Later, when I implement zenodo, there will also be keys for record_id and sha.